src/Core/Prefab/PlanarView.js

```diff
@@ -101,7 +100,7 @@ function PlanarView(viewerDiv, boundingbox, options) {
     tileLayer.update =
         processTiledGeometryNode(
             planarCulling,
-            planarSubdivisionControl(4, SSE_SUBDIVISION_THRESHOLD),
+            planarSubdivisionControl(options.maxSubdivisionLevel || 5),
```

Comment:
Maybe obvious question, but why did you change the default maxLevel from 4 to 5?